shared/common-adapters/form-with-checkbox.desktop.js

```diff
@@ -2,22 +2,25 @@
 import React, {Component} from 'react'
 import type {Props as CheckboxProps} from './checkbox'
 import type {Props} from './form-with-checkbox'
-import {Checkbox, Input, Box} from './index'
+import {Checkbox, Input, Box, Text} from './index'
 import {globalStyles} from '../styles'
 
 class FormWithCheckbox extends Component<void, Props, void> {
   render () {
     const {inputProps, checkboxesProps} = this.props
+    const {errorText = ''} = inputProps
```

Comment:
mega-🐑 

You can do:

`const {errorText = '', ...inputWOError} = inputProps`